src/pubtools/sign/bundle.py

```diff
@@ -0,0 +1,14 @@
+import click
+
+
+from .signers.msgsigner import msg_clear_sign, msg_container_sign
+
+
+@click.group()
+def cli():
+    """Pubtools-sign multi command bundle."""
+    pass
+
+
+cli.add_command(msg_clear_sign, name="msg-clear-sign")
```

Comment:
msg_clear_sign_main and msg_container_sign_main are commands.

Reply:
yeah, this needs to be rebased

Reply:
or actually not, click wrappers are around methods without main

Reply:
Isn't click on *_main functions?
https://github.com/release-engineering/pubtools-sign/blob/main/src/pubtools/sign/signers/msgsigner.py#L462
https://github.com/release-engineering/pubtools-sign/blob/main/src/pubtools/sign/signers/msgsigner.py#L478

Functions without main are not commands.